data/phmodel.txt

```diff
@@ -47,6 +47,7 @@ TRANSFORM [NH1:1]~[ND2:2]~[NH1:3]~* >> [N-:1]=[N+:2]=[N:3]-* 1E+10	# azide group
 
 #nitro group (-NO2)
 TRANSFORM [O:1]=[N:2]-[O:3] >> [O:1]=[N+:2]-[O-:3]  1E+10
+TRANSFORM [O:1]=[N:2]=[O:3] >> [O:1]=[N+:2]-[O-:3]  1E+10
```

Comment:
This one won't fly. If people have explicit hypervalent N, I don't want to suddenly force their bonding to match a different representation.